docs/zh-CN/administrator-guide/http-actions/tablets_distribution.md

```diff
@@ -0,0 +1,124 @@
+---
+{
+    "title": "GET TABLETS DISTRIBUTION BETWEEN DIFFERENT DISKS",
+    "language": "zh-CN"
+}
+---
+
+<!-- 
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# GET TABLETS DISTRIBUTION BETWEEN DIFFERENT DISKS
+   
+获取BE节点上每一个partition下的tablet在不同磁盘上的分布情况
+
+```
+curl -X GET http://be_host:webserver_port/tablets_distribution
```

Comment:
How about:
`/tablets_distribution?group_by=partition`
`/tablets_distribution?group_by=partition&partition_id=10000`

so that we can extent this api in future with some other kind of "group by"?

Reply:
> How about:
> `/tablets_distribution?group_by=partition`
> `/tablets_distribution?group_by=partition&partition_id=10000`
> 
> so that we can extent this api in future with some other kind of "group by"?

OK. I will modify the API.